jaxrs/src/main/java/org/killbill/billing/jaxrs/json/BulkBaseSubscriptionAndAddOnsJson.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.jaxrs.json;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+public class BulkBaseSubscriptionAndAddOnsJson {
+
+    @ApiModelProperty(required = true)
+    private final List<SubscriptionJson> baseEntitlementAndAddOns;
+
```

Comment:
@javier-gomez-github I would suggest to return a `201` and a `Location` header that points to the [endpoint retrieving all bundles for the account](https://github.com/killbill/killbill/blob/killbill-0.17.6/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java#L266)

Note that if we wanted to be fancy, we could add a filter query parameter for that endpoint and return that filtering in the `Location` header, so that we only return the new stuff that was created in the shopping card api. Probably not very difficult to add.